modules/rostests/apitests/imm32/clientimc.c

```diff
@@ -24,15 +24,15 @@ START_TEST(clientimc)
     DWORD dwCode;
     CLIENTIMC *pClientImc = HeapAlloc(GetProcessHeap(), HEAP_ZERO_MEMORY, sizeof(CLIENTIMC));
 
-    pClientImc->hImc = (HIMC)ImmCreateIMCC(4);
+    pClientImc->hLocalInputContext = (HLOCAL)ImmCreateIMCC(4);
```

Comment:
Just use `hInputContext` (here and below).

Reply:
OK.